tests/test_monitor.cpp

```diff
@@ -136,14 +149,36 @@ void test_monitor_versioned_basic (bind_function_t bind_function_,
     void *server = test_context_socket (ZMQ_DEALER);
 
     //  Monitor all events on client and server sockets
-    TEST_ASSERT_SUCCESS_ERRNO (zmq_socket_monitor_versioned (
-      client, "inproc://monitor-client", ZMQ_EVENT_ALL_V2, 2));
-    TEST_ASSERT_SUCCESS_ERRNO (zmq_socket_monitor_versioned (
-      server, "inproc://monitor-server", ZMQ_EVENT_ALL_V2, 2));
+    TEST_ASSERT_SUCCESS_ERRNO (zmq_socket_monitor_versioned_typed (
+      client, "inproc://monitor-client", ZMQ_EVENT_ALL_V2, 2, type_));
+    TEST_ASSERT_SUCCESS_ERRNO (zmq_socket_monitor_versioned_typed (
+      server, "inproc://monitor-server", ZMQ_EVENT_ALL_V2, 2, type_));
+
+    //  Choose the appropriate consumer socket type.
+    int mon_type;
```

Comment:
this breaks all builds:

```
[  211s] tests/test_monitor.cpp: In function 'void test_monitor_versioned_typed_basic(bind_function_t, const char*, int)':
[  211s] tests/test_monitor.cpp:173:44: error: 'mon_type' may be used uninitialized in this function [-Werror=maybe-uninitialized]
[  211s]      void *server_mon = test_context_socket (mon_type);
```

https://build.opensuse.org/package/show/network:messaging:zeromq:git-draft/libzmq

could you please send a fix?

Reply:
sure

Reply:
thanks!